RTN-003.tex

```diff
@@ -27,7 +27,7 @@
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-This document provides guidelines for groups that are independent from the LSST Project and Operations (i.e. LSST Data Facility) and would like to stand up an independent Data Access Center (IDAC; existing data centers that could serve LSST data products are considered IDACs for purposes of this document). Some IDACs may want to serve only a subset of the LSST data products: this document proposes three portion sizes, from full releases to a "light" catalog without posteriors. Guidelines and requirements for IDACs in terms of data storage, computational resources, dedicated personnel, and user authentication are described, as well as a preliminary assessment of the cost impacts. Some institutions, even those inside the US and Chile, may serve LSST data products locally to their research community. Requirements and responsibilities for such institutional bulk data transfers are also described here. \textbf{The purpose of this draft document is to serve as a \textit{preliminary} resource for partner institutions wishing to assess the feasibility of hosting an IDAC.}
+This document provides guidelines for groups that are independent from the Rubin Project and Operations (i.e. US Data Facility) and would like to stand up an independent Data Access Center (IDAC; existing data centers that could serve Rubin data products are considered IDACs for purposes of this document). Some IDACs may want to serve only a subset of the  data products: this document proposes three portion sizes, from full releases to a "lite" catalog without posteriors. Guidelines and requirements for IDACs in terms of data storage, computational resources, dedicated personnel, and user authentication are described, as well as a preliminary assessment of the cost impacts. Some institutions, even those inside the US and Chile, may serve LSST data products locally to their research community. Requirements and responsibilities for such institutional bulk data transfers are also described here.
 }
```

Comment:
I am not a fan of using the 'lite' variant of the work light. Modern use typically refers to low-calorie foods or is used in branding  In traditional (middle english) use, it means "few" or "a little"

Reply:
as discussed on slack this stays for now